src/main/java/org/broadinstitute/hellbender/utils/test/IntegrationTestSpec.java

```diff
@@ -94,17 +92,20 @@ public void executeTest(final String name, CommandLineProgramTest test) throws I
             tmpFiles.add(fl);
         }
 
-        final String args = String.format(getArgs(), tmpFiles.toArray());
+        final String preFormattedArgs = getArgs();
+        final String formattedArgs = String.format(preFormattedArgs, tmpFiles.toArray());
```

Comment:
Never mind -- just tested it and it indeed does not throw in the too-few format specifiers case (but does throw in the too-many specifiers case). Odd...